src/app/(auth)/confirm-reset-password/page.tsx

```diff
@@ -0,0 +1,21 @@
+'use client';
+
+import { useRouter } from 'next/navigation';
+import { H1, H4 } from '@/styles/text';
+import { OuterDiv, FormDiv } from '@/app/(auth)/styles';
+import BigButton from '@/components/BigButton';
+
+export default function ConfirmResetPassword() {
+  const { push } = useRouter();
+
+  return (
+    <OuterDiv>
```

Comment:
Just one remaining change! Let's get rid of the OuterDiv and FormDiv here since these are already part of the `layout.tsx`.

Reply:
gotcha!